.github/workflows/integration-test.yml

```diff
@@ -118,7 +112,7 @@ jobs:
       run: |-
         kubectl get secrets -o json | jq -r '.items[] | { name: .metadata.name, data: .data | map_values(@base64d) }'
 
-    - name: Setup tmate session
-      if: ${{ failure() }}
-      uses: mxschmitt/action-tmate@v3
-      timeout-minutes: 10
+    # - name: Setup tmate session
```

Comment:
`tmate` keeps the workflow running for a long time. 
This I believe is what caused Github to actually lock my account from using `Actions`.
So I've disabled it.

Reply:
Perhaps consider this: https://github.com/mxschmitt/action-tmate#manually-triggered-debug

Reply:
That looks good. Let me leave that for another PR since this is already so cumbersome :pensive: